### ISSUE TITLE
macOS app: Does not support categories for non-Micro.blog websites 

### ISSUE THREAD

**Issue by paulrobertlloyd** [microdotblog/issues#150]
I have set up my micropub endpoint such that it provides a list of categories (available at <https://indiekit.paulrobertlloyd.com/micropub?q=category>)

Micro.blog for iOS supports this feature:

<img src="https://user-images.githubusercontent.com/813383/61999071-75966000-b0b1-11e9-9783-ed819fbdfd87.jpeg" width="320" alt="Categories shown in the iOS app"/>

However, Micro.blog for macOS appears to only support showing categories for hosted micro blogs. Is this intentional?

**Comment by manton** [comment]
I don't think that was intentional. Thanks for reporting it! (Also, just in case it's not clear, the macOS app hides the categories by default… You can show them in the View menu.)

**Comment by paulrobertlloyd** [comment]
@manton Good to hear! I wasn’t sure so that’s why I tested this feature with a hosted blog, where in that instance, they showed up.

Also… I’ve currently set up my micropub endpoint to list all the categories I’ve ever used on my blog (perhaps not a great idea, but hey). This list currently runs to ~250 categories! That is a bit excessive, but while the view in the iOS app handles that well enough, not sure the way they are currently laid out in the macOS app would fare as well! ☺️

**Comment by manton** [comment]
Thanks @paulrobertlloyd, reviewing this again… So the problem is that there are so many categories that the UI can't adapt to show them, right, not that Micro.blog isn't downloading the categories from Micropub at all?

**Comment by paulrobertlloyd** [comment]
Well, it may very well be both! Currently, I can’t see any categories at all. I just tried changing my Micropub config to provide only 3 categories, and still no categories appeared.
